carina-webdriver/src/main/java/com/qaprosoft/carina/core/foundation/utils/mobile/MobileUtils.java

```diff
@@ -721,6 +723,20 @@ public static void zoom(Zoom type) {
             break;
         }
     }
+
+    public static boolean isAppRunning() {
```

Comment:
@jcanavire-solvd, could you please port this method into IMobileUtils as well.
current class should be removed soon.
Thanks a lot.